comms/src/peer_manager/mod.rs

```diff
@@ -20,11 +20,9 @@
 //  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 //  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-pub mod ban_list;
 pub mod liveness;
-pub mod manager;
 pub mod node_id;
 pub mod node_identity;
 pub mod peer;
-pub mod routing_table;
-pub mod storage;
+pub mod peer_manager;
```

Comment:
Suggestion for the future: Something I would consider doing in the `mod.rs` files is re-exporting the commonly used structs from inside this module. For example you put here:
```pub use self::node_id::NodeId; ```
Then when importing it elsewhere instead of saying:
```use crate::peer_manager::node_id::NodeId;```
they can just say:
```use crate::peer_manager::NodeId;```
It means that user's of this module don't have to dig around in the sub files to find the struct they need.